pkg/controller/jaeger/elasticsearch.go

```diff
@@ -22,6 +22,7 @@ func (r *ReconcileJaeger) applyElasticsearches(jaeger v1.Jaeger, desired []esv1.
 		"app.kubernetes.io/instance": jaeger.Name,
 		"app.kubernetes.io/part-of":  "jaeger",
 	})
+	opts = client.InNamespace(jaeger.Namespace)
```

Comment:
Why changing the query labels? If it is required the previous initialization should be removed.

Reply:
I think this is a leftover from a test.

Reply:
Fixed.